src/contribution-guide/suggesting-an-improvement.md

```diff
@@ -0,0 +1,33 @@
+
+In most cases, any new documentation work for Kubewarden begins with an issue in GitHub. The [documentation team for Kubewarden](https://github.com/orgs/kubewarden/teams/kubewarden-documentation) reviews, categorizes and tags them as appropriate. Everybody is welcome to work on the issue, including the reporter, but we request that you assign it to yourself before commencing any work to avoid duplicate efforts.
+
+## Opening an issue
+
+If you want to suggest improvements to existing documentation content or notice an error, then open an issue.
+
+- Click the GitHub Octocat icon on the top. This redirects you to the [documentation repository for Kubewarden](https://github.com/kubewarden/docs).
+- Navigate to the Issues tab and click **New issue**.
+- Describe the issue or suggestion for improvement. The more details you provide, the better!
+- Click Submit new issue.
+- Post submitting, you can either assign it to yourself or wait for a community member to pick it up. Members of the documentation team and from the community might request clarifications before they can take action on your issue, so we'd request you to actively check your issue or [turn on GitHub notifications](https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications).
+
+## New content suggestions
+
+If you want to suggest new content, please file an issue following the steps above. Either
+
+- You can choose an existing page in the section you think the content belongs in and click Create an issue.
+OR
+- Navigate to [GitHub](https://github.com/kubewarden/docs/issues/new/choose) and file the issue directly.
+
+## How do I make my contribution count?
+
+No contribution is big or small! However, to ensure that the community derives maximum value we'd request you to follow the below when reporting an issue:
+
+- Focus on providing a clear description of the issue. Some key points to consider would be specifically describing what is missing, outdated, erroneous, or requires qualitative/technical improvement.
+- Detailing the specific impact the issue has on users.
+- Delimiting the scope of the issue. If the scope is larger, we'd request you to break it down to smaller tasks within an issue. For example, "Creating a Contribution guide" is very widely scoped since there would be multiple tasks associated with the issue. However, "Fixing grammatical error on the Quickstart page" is a more narrowly scoped issue that would, potentially, require only a single pull request.
+- Crosscheck existing issues to avoid duplicate work.
+- There is a fair chance that you're opening an issue with reference to another pull request or existing issue. Ensure you reference it in the issue you're opening to provide context for contributors who may want to work on it.
```

Comment:
```suggestion
- Check for an existing pull request or issue as there is a fair chance that you're opening an issue that has been referenced before. Ensure you reference the existing issue or pull request within the issue you're opening to provide context for contributors who may want to work on it.
```

Reply:
For these, suggesting to make the sentences parallel with common verb beginnings and tenses for easier comprehension.